sdk/core/src/shared/getOptionsFromEnvironment.ts

```diff
@@ -0,0 +1,116 @@
+import { KeysService } from '@affinidi/common'
+import { Env, SdkOptions } from '../dto/shared.dto'
+
+import {
+  DEV_REVOCATION_URL,
+  PROD_REVOCATION_URL,
+  STAGING_REVOCATION_URL,
+  DEV_COGNITO_CLIENT_ID,
+  DEV_COGNITO_USER_POOL_ID,
+  DEV_REGISTRY_URL,
+  DEV_ISSUER_URL,
+  DEV_VERIFIER_URL,
+  DEV_KEY_STORAGE_URL,
+  DEV_VAULT_URL,
+  DEV_PHONE_ISSUER_BASE_PATH,
+  DEV_EMAIL_ISSUER_BASE_PATH,
+  STAGING_COGNITO_CLIENT_ID,
+  STAGING_COGNITO_USER_POOL_ID,
+  STAGING_REGISTRY_URL,
+  STAGING_ISSUER_URL,
+  STAGING_VERIFIER_URL,
+  STAGING_KEY_STORAGE_URL,
+  STAGING_VAULT_URL,
+  STAGING_PHONE_ISSUER_BASE_PATH,
+  STAGING_EMAIL_ISSUER_BASE_PATH,
+  PROD_COGNITO_CLIENT_ID,
+  PROD_COGNITO_USER_POOL_ID,
+  PROD_REGISTRY_URL,
+  PROD_ISSUER_URL,
+  PROD_VERIFIER_URL,
+  PROD_KEY_STORAGE_URL,
+  PROD_VAULT_URL,
+  PROD_PHONE_ISSUER_BASE_PATH,
+  PROD_EMAIL_ISSUER_BASE_PATH,
+  DEV_METRICS_URL,
+  STAGING_METRICS_URL,
+  PROD_METRICS_URL,
+} from '../_defaultConfig'
+
+export const getAccessApiKeyFromOptions = (options: SdkOptions) => {
+  if (options.accessApiKey && !options.apiKey) {
+    return options.accessApiKey
+  }
+
+  const apiKey = options.apiKey || 'testApiKey'
```

Comment:
@inga-affinidi From which part you made this copy-paste ? , should be NO default value here

Reply:
@DenisPopov15 , it is a refactored version of `CommonNetworkMember._setAccessApiKey`: https://github.com/affinityproject/affinidi-core-sdk/blob/2d2bc095a161b1e6fa6b489181bbc6c78d849060/sdk/core/src/CommonNetworkMember.ts#L220-L243

Reply:
ok, I see, then its still left there, this thing is obsolete and we removed this default key which was originally done for softer transaction from allow keys at key middleware a while ago and finished that transaction (so this default key not working anymore). So please remove it, and make validation/check, that apiKey or accessApiKey should be required/provided. Thnx

Reply:
OK

Reply:
Fixed